src/literals.py

```diff
@@ -15,3 +15,8 @@
 UI_FUNCTIONS = ["app", "app-gunicorn"]
 DEFAULT_ROLES = ["Public", "Gamma", "Alpha", "Admin"]
 SQL_AB_ROLE = "SELECT name FROM ab_role;"
+
+# Observability literals
+LOG_FILE = "/var/log/superset.log"
+EXPOSE_PORT = 9102
+ACCEPT_PORT = 9125
```

Comment:
Could we improve the variable naming here to make it clearer? Maybe `STATSD_PORT` instead of `ACCEPT_PORT` and `PROMETHEUS_METRICS_PORT` instead of `EXPOSE_PORT`?